### PR TITLE
Adds Blueshield Beret to Ian clothing

### PR DESCRIPTION
**Pull request by Ralta**
Allows Ian to wear the Officer's Beret and the Navy Blue Officer's Beret in order to become Blueshield Ian, protector of Command and biter of ankles.

Original PR was by ComradeCheekiBreeki on Oracle here:

https://github.com/OracleStation/OracleStation/pull/1195

Added the sprite for Blueshield's armor vest - Ian can now wear this and look like a badass the whole time:
Ian's Blueshield Outfit
Examine text

:cl:
add: Ian can now wear a CentComm Officer Beret to become Blueshield Ian
add: Sprite added for Blueshield armor to corgi_back.dmi
tweak: beret_centcom_officer and beret_centcom_officer_navy copied to corgi_head.dmi spritesheet
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
Sounds like a partial port of OracleStation/OracleStation#1195 if the link is okay to make. I know this because I was thinking of doing this myself some time eventually. If we have the blueshield armor too you think Ian could put that on as well? Or does he only get hats on Paradise, I can't remember

**Comment by Ralta** [comment]
> Sounds like a partial port of [OracleStation/OracleStation#1195](https://github.com/OracleStation/OracleStation/pull/1195) if the link is okay to make. I know this because I was thinking of doing this myself some time eventually. If we have the blueshield armor too you think Ian could put that on as well? Or does he only get hats on Paradise, I can't remember

Yeah it is literally from there - I meant to credit Cheeki actually and completely forgot!

Ian can already wear armor, including the Blueshields I believe. I just figured wearing the beret should be enough to make him an honorary Blueshield.

File: html/changelogs/ianshield.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+author: Ralta
+
+delete-after: True
+
+changes: 
+  - rscadd: "Added ability for Ian to wear Blueshield/CentComm berets."
```
Comment (Fox-McCloud) [inline html/changelogs/ianshield.yml]: Don't do this; we haven't done changelogs like this in years.

Comment (Ralta) [reply html/changelogs/ianshield.yml]: Dangit, there was me thinking I was following the correct process! :joy:

Comment (taukausanake) [reply html/changelogs/ianshield.yml]: It's okay, I did it too a long time ago. Almost got away with it too :P

**Comment by Spacemanspark** [comment]
Can you add the deathsquad armor and helmet to Ian? /tg/ has it and I tried to do it years ago but failed

**Comment by Ralta** [comment]
> Can you add the deathsquad armor and helmet to Ian? /tg/ has it and I tried to do it years ago but failed

Deathsquad armour and helmet are already there, including sprites 😉 

```
if(/obj/item/clothing/head/helmet/space/deathsquad)
	name = "Trooper [real_name]"
	desc = "That's not red paint. That's real corgi blood."
	valid = 1
```

**Pull request merged**